src/engine/loopingcontrol.cpp

```diff
@@ -94,13 +116,26 @@ LoopingControl::LoopingControl(QString group,
     m_pTrackSamples = ControlObject::getControl(ConfigKey(group, "track_samples"));
     m_pSlipEnabled = ControlObject::getControl(ConfigKey(group, "slip_enabled"));
 
-    // Connect beatloop, which can flexibly handle different values.
-    // Using this CO directly is meant to be used internally and by scripts,
-    // or anything else that can pass in arbitrary values.
+    // DEPRECATED: Use beatloop_size and beatloop_toggle instead.
+    // Activates a beatloop of a specified number of beats.
     m_pCOBeatLoop = new ControlObject(ConfigKey(group, "beatloop"), false);
     connect(m_pCOBeatLoop, SIGNAL(valueChanged(double)), this,
             SLOT(slotBeatLoop(double)), Qt::DirectConnection);
 
+    m_pCOBeatLoopSize = new ControlObject(ConfigKey(group, "beatloop_size"),
+                                          true, false, false, 4.0);
+    m_pCOBeatLoopSize->connectValueChangeRequest(this,
+            SLOT(slotBeatLoopSizeChangeRequest(double)), Qt::DirectConnection);
+    m_pCOBeatLoopToggle = new ControlPushButton(ConfigKey(group, "beatloop_toggle"));
+    connect(m_pCOBeatLoopToggle, SIGNAL(valueChanged(double)),
+            this, SLOT(slotBeatLoopToggle(double)));
```

Comment:
No, it is there. I changed #940 to use this instead of loopauto_toggle. Maybe you need to pull the updated version of #940.